git-tar/function/handler.go

```diff
@@ -36,6 +37,11 @@ func Handle(req []byte) []byte {
 		os.Exit(-1)
 	}
 
+	if ok, errMessage := isWithinLimit(len(stack.Functions)); !ok {
```

Comment:
This looks great, but we need to do a little more before releasing.

This should communicate back via audit and also via GitHub statuses. As it is now the user won't get any feedback.

Reply:
Ok, I will add that.

Reply:
@alexellis I think @s8sg has already added most of the code required to add git-status in PR #63. I think it makes more sense to make me these changes once his PR gets merged.